idaes/models/unit_models/tests/test_separator.py

```diff
@@ -664,6 +665,49 @@ def test_add_material_splitting_constraints_none_phase_component(self, build):
 
         assert not hasattr(build.fs.sep, "material_splitting_eqn")
 
+    @pytest.mark.unit
+    def test_add_momentum_splitting_constraints_none_total(self, build):
+        build.fs.sep.config.momentum_balance_type = MomentumBalanceType.none
+
+        build.fs.sep.add_split_fractions(build.outlet_list, build.fs.sep.mixed_state)
+
+        build.fs.sep.add_material_splitting_constraints(build.fs.sep.mixed_state)
```

Comment:
This should be calling `add_momentum_splitting_constraints`. Same applies below.

However, to make this even more thorough (and less repetitive), I suggest using a parameterized test:

```
@pytest.mark.parametrize("balance_type", MomemtumBalanceType)
@pytest.mark.parametrize("split_type", SplittingType)
@pytest.mark.unit
def test_momentum_splitting(self, build, balance_type, split_type):
    build.fs.sep.config.split_basis = split_type
    build.fs.sep.config.momentum_balance_type = balance_type

    build.fs.sep.add_split_fractions(build.outlet_list, build.fs.sep.mixed_state)
    build.fs.sep.add_momentum_splitting_constraints(build.fs.sep.mixed_state)

    if balance_type is MomentumBalanceType.none:
        assert not hasattr(build.fs.sep, "momentum_splitting_eqn")
    else:
        assert isinstance(build.fs.sep.momentum_splitting_eqn, Constraint)
```

This will cover all combinations in a single test and can replace all of your new tests and the existing one in one neat block of code.

Reply:
Rewrote accordingly